config/kafka-instance-types-configuration.yaml

```diff
@@ -48,3 +48,17 @@ supported_instance_types:
       quotaConsumed: 1
       quotaType: "rhosak"
       capacityConsumed: 1
+  - id: eval
+    display_name: Eval
+    sizes:
+    - id: x1
+      ingressThroughputPerSec: "30Mi"
+      egressThroughputPerSec: "30Mi"
+      totalMaxConnections: 3000
+      maxConnectionAttemptsPerSec: 100
+      maxDataRetentionSize: "1000Gi"
+      maxDataRetentionPeriod: "P14D"
+      maxPartitions: 1000
+      quotaConsumed: 1
```

Comment:
`quotaConsumed` should be 0

Reply:
I think it does not really matter at this point as no kafka eval instances can be created right? or would existing instances be updated?

Reply:
this is used in `ReserveQuota` which is only called when creating an instance. @ziccardi can you confirm?

Reply:
@ziccardi quotyConsumed is used: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/larger-kafka-instances/internal/kafka/internal/services/quota/ams_quota_service.go#L136

Reply:
so we need to use the correct values.

Reply:
Values updated. For details see the new comments in the issue